io/net/ethtool_test.py

```diff
@@ -54,10 +54,14 @@ def setUp(self):
             if not smm.check_installed(pkg) and not smm.install(pkg):
                 self.cancel("%s package is need to test" % pkg)
         interfaces = netifaces.interfaces()
-        interface = self.params.get("interface")
-        if interface not in interfaces:
-            self.cancel("%s interface is not available" % interface)
-        self.iface = interface
+        local = LocalHost()
```

Comment:
even i feel if we pass mac address we can eliminate the dependency of  `netifaces` as well

Reply:
why so I still feel it is the primary way as it never changes in general cases like as it is driven by h/w properties